### PR TITLE
Serial Volume Access - etcd

### PR DESCRIPTION
**Pull request by akutz**
This patch adds etcd support to the serial volume access middleware, meaning the middleware can now support distributed locking.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -162,17 +167,24 @@ test:  build
 endif
 
 # Because Travis-CI's containers have limited resources, the Mock SP's
-# idempotency provider's timeout needs to be increased from the default
-# value of 0 to 1s. This ensures that lack of system resources will not
-# prevent a single, non-concurrent RPC from failing due to an OpPending
-# error.
+# serail volume access provider's timeout needs to be increased from the
```
Comment (codenrhoden) [inline Makefile]: s/serial/serail

File: middleware/serialvolume/etcd/etcd_lock_provider.go [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+		}
+	}
+}
+
+// Unlock unlocks m. It is a run-time error if m is not locked on entry to
+// Unlock.
+//
+// A locked TryMutex is not associated with a particular goroutine. It is
+// allowed for one goroutine to lock a Mutex and then arrange for another
+// goroutine to unlock im.
```
Comment (codenrhoden) [inline middleware/serialvolume/etcd/etcd_lock_provider.go]: unlock "it" or "m" ?

Comment (akutz) [reply middleware/serialvolume/etcd/etcd_lock_provider.go]: "it" and fixed

File: Makefile (hunk 1 above)
Comment (akutz) [reply Makefile]: fixed

**Pull request merged**